src/azure-cli/azure/cli/command_modules/profile/_validators.py

```diff
@@ -0,0 +1,38 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import uuid
+
+from azure.cli.core.util import should_disable_connection_verify
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def _is_guid(guid):
+    try:
+        uuid.UUID(guid)
+        return True
+    except ValueError:
+        pass
```

Comment:
you can `return False` here

Reply:
Good point! This is actually copied from https://github.com/Azure/azure-cli/blob/fa3258bb1de15383e4c9568f039ff2440972c5c3/src/azure-cli/azure/cli/command_modules/role/custom.py#L1659 and other places. I will fix this by now.